examples/demo_threading/images/kdabLogo.png.license

```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Nuno Pinheiro <nuno@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
Stating MIT or Apache-2.0 here might be an issue, this is the KDAB logo, after all :thinking: 
We do *not* want to hand this out freely.

Reply:
Maybe we should make this a new repository that we take in as a submodule a closed license :thinking:

Reply:
Note that you can create custom licenses in the SPDX format, i've done this in other repositories

Reply:
Need to figure out the license of this one :sweat_smile:

Reply:
Done